buddypress/groups/groups-loop.php

```diff
@@ -28,7 +28,7 @@
 			bp_the_group();
 			?>
 
-            <li class="nhsuk-grid-column-one-third nhsuk-card-group__item">
+            <li class="nhsuk-grid-column-one-third nhsuk-card-group__item"  data-bp-item-id="<?php bp_group_id(); ?>" data-bp-item-component="groups">
```

Comment:
group id to item id? is that right?

Reply:
Yes. It is a BB inherited requirement to trigger the correct js responses in content.